autowrap/PXDParser.py

```
@@ -354,8 +363,13 @@ def parseTree(cls, node: CEnumDefNode, lines, pxd_path):
             scoped = node.scoped
         except AttributeError:
             scoped = False
-        annotations = parse_class_annotations(node, lines)
+        try:
+            annotations = parse_class_annotations(node, lines)
+        except ValueError as e:
+            logger.warning(" Failed to parse annotations in %s:%d\n%s", pxd_path, node.pos[1], e)
+            raise ValueError(f"Failed to parse annotations in {pxd_path}:{node.pos[1]}\n{e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)